src/main.rs

```diff
@@ -339,6 +353,7 @@ fn merge_avgs(
             severity: avg_severity,
             status: avg_status,
             required_by: vec![],
+            avg_types: avg_types.iter().cloned().collect(),
```

Comment:
```suggestion
            avg_types: avg_types.into_iter().collect(),
```
should be sufficient